packages/insight/src/components/search.tsx

```diff
@@ -34,6 +34,14 @@ const SearchForm = styled.form<{ borderBottom?: boolean }>`
     borderBottom ? `1px solid ${colors.borderColor}` : 'none'};
 `;
 
+const SearchImg = styled.img`
+  alt: search;
```

Comment:
I don't think you can add an `alt` tag in css, I would add this back into the html:
```
<SearchImg src={searchIcon} alt="search" />
```